lib/commands/setup.js

```diff
@@ -111,7 +111,7 @@ class SetupCommand extends Command {
             }
         }];
 
-        if (!argv.local && argv['setup-linux-user'] !== false && os.platform() === 'linux') {
+        if (!argv.local && argv['setup-linux-user'] !== false && os.platform() === 'linux' && argv.process !== 'local') {
```

Comment:
If `argv.process == local`, do we need to check `argv.local`?

Reply:
probably not

Reply:
will leave it in there though - doesn't really hurt anything and it's good just to make sure 😄